python/chronos/script/test-run.py

```diff
@@ -0,0 +1,110 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.chronos.forecaster.tcn_forecaster import TCNForecaster 
+import pandas as pd 
+from bigdl.chronos.data import TSDataset
+from bigdl.chronos.data.repo_dataset import get_public_dataset
+from sklearn.preprocessing import StandardScaler 
+from bigdl.chronos.metric.forecast_metrics import Evaluator  
+import torch
+import time 
+import numpy as np 
+
+look_back = 96 
+horizon = 48
+
+def generate_data(dataset_name: str = "nyc_taxi"): 
+    if dataset_name == "nyc_taxi":
+        tsdata_train, tsdata_val, tsdata_test = get_public_dataset(name='nyc_taxi', with_split=True,
+                                                                   val_ratio=0.1, test_ratio=0.1)
+    else:
+        np.random.seed(0) 
+        df = pd.DataFrame(np.random.randint(1, 1000, size=(5000, 1)), columns=["target"])
+        df.insert(0, "date", pd.date_range(start="2022-7-22", periods=5000, freq="D"))
+        tsdata_train, tsdata_val, tsdata_test = TSDataset.from_pandas(df, dt_col="date", target_col="target",
+                                                                      with_split=True, test_ratio=0.1, val_ratio=0.1)
+
+    standard_scaler = StandardScaler() 
+    
+    if dataset_name == "nyc_taxi":
+        for tsdata in [tsdata_train, tsdata_val, tsdata_test]:
+            tsdata.gen_dt_feature(one_hot_features=['HOUR'])\
+                  .impute(mode="last")\
+                  .scale(standard_scaler, fit=(tsdata is tsdata_train)) 
+    else:
+        for tsdata in [tsdata_train, tsdata_val, tsdata_test]:
+            tsdata.gen_dt_feature(one_hot_features=['date'])\
+                  .impute(mode="last")\
+                  .scale(standard_scaler, fit=(tsdata is tsdata_train)) 
+
+    train_loader = tsdata_train.to_torch_data_loader(roll=True, lookback=look_back, horizon=horizon)
+    test_loader = tsdata_test.to_torch_data_loader(batch_size=1, roll=True, lookback=look_back, horizon=horizon, shuffle=False) 
+
+    return train_loader, test_loader
+
+
+
+def test_run(dataset_name: str = "nyc_taxi"):
+    train_loader, test_loader = generate_data(str)
+    forecaster = TCNForecaster(past_seq_len = look_back,   
+                            future_seq_len = horizon,
+                            input_feature_num = 8,   
+                            output_feature_num = 1,   
+                            num_channels = [30] * 7, 
+                            repo_initialization = False, 
+                            kernel_size = 3, 
+                            dropout = 0.1, 
+                            lr = 0.001,
+                            seed = 1)
```

Comment:
no need to set hyperparameters after `output_feature_num`

Reply:
use https://bigdl.readthedocs.io/en/latest/doc/PythonAPI/Chronos/forecasters.html#bigdl.chronos.forecaster.tcn_forecaster.TCNForecaster.from_tsdataset this API